versions.tf

```diff
@@ -1,3 +1,10 @@
 terraform {
   required_version = ">= 0.14"
+  required_providers {
```

Comment:
Remove the required providers here. This module is compatible with all versions of google so far